sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -668,7 +675,7 @@ class SQLContext private[sql](val sparkSession: SparkSession)
    * @since 1.3.0
    */
   def tables(): DataFrame = {
-    Dataset.ofRows(sparkSession, ShowTablesCommand(None, None))
+    Dataset.ofRows(sparkSession, ShowTablesCommand(None, None, defaultTablesOutput))
```

Comment:
how about we create `ShowTables(UnresolvedNamespace(Nil), None)` here?

Reply:
Good idea.